src/OnboardingSPA/steps/TheFork/stylesheet.scss

```diff
@@ -51,6 +51,8 @@
 				text-align: center;
 				margin: 25px !important;
 				color: var(--nfd-onboarding-primary);
+				text-decoration: underline;
+				cursor: pointer;
```

Comment:
Can we can remove this once we consolidate the logic to the `handleMigration` function?